R/0_generics.R

```diff
@@ -294,3 +294,29 @@ ff_starters <- function(conn, ...) {
 ff_starters.default <- function(conn, ...) {
   stop(glue::glue("No method of ff_starters found for platform: {conn$platform}."))
 }
+
+#' @export
+ff_playerscores.default <- function(conn, season, week, ...) {
+  stop(glue::glue("No method of ff_playerscores found for platform: {conn$platform}."))
+}
+
+## ff_scoringhistory ##
+
+#' Get a dataframe of scoring history, utilizing the ff_scoring and load_player_stats functions.
+#'
+#' @param conn a conn object created by \code{ff_connect()}
+#' @param season the season of interest back to 1999
```

Comment:
Is there a benefit to having season parameter? Can we set default = NULL and do an optional filter?

Reply:
Resolved with default numeric :)

Reply:
Hmm. Actually, can you update the docs to say 
```
@param season a numeric vector of seasons (earliest available year is 1999)
```